src/Standards/PSR12/Docs/ControlStructures/ControlStructureSpacingStandard.xml

```diff
@@ -0,0 +1,124 @@
+<documentation title="Control Structure Spacing">
+    <standard>
+    <![CDATA[
+    The control structures must have one space after the control structure keyword, no spaces after the opening parenthesis and before the closing parenthesis, and one space between the closing parenthesis and the opening brace.
+    ]]>
+    </standard>
+    <code_comparison>
+        <code title="Valid: No space after the opening parenthesis in a single-line condition.">
+        <![CDATA[
+if <em>(</em>$expr) {
+}
+        ]]>
+        </code>
+        <code title="Invalid: Space after the opening parenthesis in a single-line condition.">
+        <![CDATA[
+if <em>( </em>$expr) {
+}
+        ]]>
+        </code>
+    </code_comparison>
+    <code_comparison>
+        <code title="Valid: No space before the closing parenthesis in a single-line condition.">
+        <![CDATA[
+if <em>(</em>$expr) {
+}
+        ]]>
+        </code>
+        <code title="Invalid: Space before the closing parenthesis in a single-line condition.">
+        <![CDATA[
+if ($expr<em> </em>) {
+}
+        ]]>
+        </code>
+    </code_comparison>
+    <standard>
+    <![CDATA[
+    The body of the multi-line control structure must be indented once, placing the body on the next line after the opening brace.
```

Comment:
This is not about the body of a multi-line control structure, but about multi-line control structure conditions.

The body of a control structure is what is between the curly braces...

Reply:
Yes, but the docs are mentioning the _structure_ body, which I thought was the 'body' of the conditions that are multi-lined.